lasagne/layers/dense.py

```diff
@@ -129,17 +132,21 @@ def get_output_shape_for(self, input_shape):
         return (input_shape[0], self.num_units) + input_shape[2:]
 
     def get_output_for(self, input, *args, **kwargs):
-        out_r = T.tensordot(self.W, input, axes=[[0], [1]]) # cf * bc01... = fb01...
-        remaining_dims = range(2, input.ndim) # input dims to broadcast over
-        out = out_r.dimshuffle(1, 0, *remaining_dims) # bf01...
+        # cf * bc01... = fb01...
+        out_r = T.tensordot(self.W, input, axes=[[0], [1]])
+        # input dims to broadcast over
+        remaining_dims = range(2, input.ndim)
+        # bf01...
+        out = out_r.dimshuffle(1, 0, *remaining_dims)
 
         if self.b is None:
             activation = out
         else:
             if self.untie_biases:
-                remaining_dims_biases = range(1, input.ndim - 1) # no broadcast
+                # no broadcast
+                remaining_dims_biases = range(1, input.ndim - 1)
             else:
-                remaining_dims_biases = ['x'] * (input.ndim - 2) # broadcast
+                remaining_dims_biases = ['x'] * (input.ndim - 2)  # broadcast
```

Comment:
Yeah, that was because `# no broadcast` is too long and `# broadcast` isn't.  I can change this.